test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -666,6 +666,23 @@ TEST_F(AtenXlaTensorTest, TestQR) {
   }
 }
 
+TEST_F(AtenXlaTensorTest, TestSymEig) {
+  static const int dims[] = {4, 7};
+  for (auto m : dims) {
+    at::Tensor a = at::rand({m, m}, at::TensorOptions(at::kFloat));
+    at::Tensor sym_a = a.mm(a.t());
+    auto b = at::symeig(sym_a, /*eigenvectors=*/true, /*upper=*/true);
```

Comment:
I'd test for other combinations as well, nesting loops like:
```
for (upper : {true, false})
  for (eigenvectors : {true, false})
```

Reply:
Sure, I'll add that. Thank you.